cmd/kfulint/testdata/long-chain/checker_tests.go

```diff
@@ -0,0 +1,27 @@
+package checker_test
+
+type t struct {
+	a struct {
+		b struct {
+			c struct {
+				d struct {
+					e struct {
+						p1, p2, p3, p4 int
+					}
+				}
+			}
+		}
+	}
+}
+
+func sample() {
```

Comment:
Can you add samples that do not trigger a warning?
For example, chains with `n=2` and `n=3`.